### PR TITLE
Grant Thoth Station team write access on the tutorial repos

### PR DESCRIPTION
**Pull request by pacospace**
Signed-off-by: Francesco Murdaca <fmurdaca@redhat.com>

### CODE REVIEW

File: github-config.yaml [hunk 1]
```diff
@@ -595,7 +595,6 @@ orgs:
           learn-katacoda: admin
           log-anomaly-detector: admin
           ludus: admin
-          manage-dependencies-tutorial: admin
```
Comment (goern) [inline github-config.yaml:598]: @pacospace please dont remove them from the SourceOps team, as the bots still need to be maintainers, as the assist us humans.

**Review by goern — CHANGES_REQUESTED** [review]

**Comment by pacospace** [comment]
> /assign @pacospace

updated! Thanks!

**Pull request merged**